docker/Dockerfile

```diff
@@ -15,4 +15,8 @@ RUN apk add --no-cache ca-certificates fuse openssh-client tzdata jq
 
 COPY --from=builder /go/src/github.com/restic/restic/restic /usr/bin
 
-ENTRYPOINT ["/usr/bin/restic"]
+COPY docker/entrypoint.sh /
+
+VOLUME ["/data"]
+
+ENTRYPOINT ["/entrypoint.sh"]
```

Comment:
This changes the default behavior from printing a help message to actually performing a backup. It won't make much of a difference when your setup is already working, but might be unexpected for new users.